### DISCUSSION TITLE
Security Question!

### DISCUSSION THREAD

**Discussion by torerik** [enkelmedia/TheDashboard#93] [General]
First - i really like this feature, its a good feature if there are multiple editors and so on :)
But i have a question about the security, before i do my own test, is it possible to reach the api calls from outside. i think about the sql calls and so on?

**Comment by enkelmedia** [comment]
Hi!

Thanks a lot! Nice to hear that you like it =D

The package uses the [UmbracoAuthorizedJsonController](https://docs.umbraco.com/umbraco-cms/implementation/controllers#backoffice-authorization) which is protected by the default Umbraco login which means that only logged-in backoffice-users would be able to call the APIs.

You can have a look at the source code here on github to get a detailed idea of how it works.

**Comment by torerik** [comment]
Hi again!
I have an security issue with umbraco cloud and TheDashboard from
enkelmedia.
[image: image.png]

Is this something you know about - what to do?

On Thu, Sep 14, 2023 at 10:01 AM Enkel Media ***@***.***>
wrote:

> Closed #93 <https://github.com/canarytrace/documentation/discussions/93>
> as resolved.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/enkelmedia/TheDashboard/discussions/93>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AACO4QO5GYTWJSCYFGZHOCTX2K2WHANCNFSM6AAAAAA4XT5DB4>
> .
> You are receiving this because you authored the thread.Message ID:
> <enkelmedia/TheDashboard/repo-discussions/93/discussion_event/857509@
> github.com>
>


-- 

Tor Erik Skogen
Designer / Utvikler
98895985

**Comment by enkelmedia** [comment]
Hi @torerik 

Your image was not included in the post - could you explain a little more?

**Comment by torerik** [comment]
Here's the image :) Thanks for answer

On Mon, Dec 11, 2023 at 3:06 PM Enkel Media ***@***.***>
wrote:

> Hi @torerik <https://github.com/torerik>
>
> Your image was not included in the post - could you explain a little more?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/enkelmedia/TheDashboard/discussions/93#discussioncomment-7820245>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AACO4QNYKHF7RW52HMS5553YI4HPXAVCNFSM6AAAAAA4XT5DB6VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TQMRQGI2DK>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Tor Erik Skogen
Designer / Utvikler
98895985

**Comment by enkelmedia** [comment]
@torerik Still no image, I would recommend that you write answers to the issues on github and not over email - might help.

**Comment by torerik** [comment]
Attached files?
Ok, it was som esecurity issue with identity deploy on ubraco cloud.

Latest upgrade has failed - Error restoring packages: Determining projects
to restore... Restored
/app/work/repository/src/UmbracoProject/Umbraco Project.csproj (in 2.26
min).
/app/work/repository/src/Our.Umbraco.TheDashboard/Our.Umbraco.TheDashboard.csproj
: error NU1903: Warning As Error: Package 'Umbraco.Cms.Web.BackOffice'
11.0.0 has a known high severity vulnerability,
https://github.com/advisories/GHSA-h8wc-r4jh-mg7m
[/app/work/repository/src/Umbraco Project/Umbraco Project.csproj] Failed to
restore
/app/work/repository/src/Our.Umbraco.TheDashboard/Our.Umbraco.TheDashboard.csproj
(in 150 ms).

On Tue, Dec 12, 2023 at 11:46 AM Enkel Media ***@***.***>
wrote:

> @torerik <https://github.com/torerik> Still no image, I would recommend
> that you write answers to the issues on github and not over email - might
> help.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/enkelmedia/TheDashboard/discussions/93#discussioncomment-7829525>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AACO4QNHKZ37LDMHBLMGHSDYJAYYHAVCNFSM6AAAAAA4XT5DB6VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TQMRZGUZDK>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Tor Erik Skogen
Designer / Utvikler
98895985

**Comment by enkelmedia** [comment]
It looks like you have a local copy of the package source code? I can't really help you with that.

`/app/work/repository/src/Our.Umbraco.TheDashboard/Our.Umbraco.TheDashboard.csproj` is not something that I recognize.

Also, the dependencies the package has on Umbraco is "the lowest version" it supports so as long as you keep your Umbraco-version up to date it should not be a problem.

**Comment by torerik** [comment]
Ok, thanks for the clarification.

On Thu, Dec 14, 2023 at 11:47 PM Enkel Media ***@***.***>
wrote:

> It looks like you have a local copy of the package source code? I can't
> really help you with that.
>
>
> /app/work/repository/src/Our.Umbraco.TheDashboard/Our.Umbraco.TheDashboard.csproj
> is not something that I recognize.
>
> Also, the dependencies the package has on Umbraco is "the lowest version"
> it supports so as long as you keep your Umbraco-version up to date it
> should not be a problem.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/enkelmedia/TheDashboard/discussions/93#discussioncomment-7858501>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AACO4QKI5ARL4WFYQ7JJ7JDYJN6YXAVCNFSM6AAAAAA4XT5DB6VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TQNJYGUYDC>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Tor Erik Skogen
Designer / Utvikler
98895985
